mep3_simulation/resource/ros2_control_configuration.yml

```
@@ -0,0 +1,20 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 50.0
+
+    diffdrive_controller:
+      type: diff_drive_controller/DiffDriveController
+
+    joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+
+diffdrive_controller:
+  ros__parameters:
+    left_wheel_names: ["motor_l"]
+    right_wheel_names: ["motor_r"]
```

```suggestion
    left_wheel_names: ["motor_l"]
    right_wheel_names: ["motor_r"]
```
Imena motora treba izmijeniti (ovdje, URDF i Webots) tako da budu ista kao i u `mep3_driver`. Cilj je da mep3_driver i mep3_simulation dijele istu konfiguraciju (ali to polako :))